src/sinks/humio_logs.rs

```diff
@@ -65,7 +65,12 @@ impl SinkConfig for HumioLogsConfig {
 
 impl HumioLogsConfig {
     fn build_hec_config(&self) -> HecSinkConfig {
-        let host = self.host.clone().unwrap_or_else(|| HOST.to_string());
+        let mut host = self.host.clone().unwrap_or_else(|| HOST.to_string());
+
+        if host.ends_with('/') {
+            host.pop();
+        }
+
```

Comment:
Could this be done just by adding a `.trim_end_matches('/').to_string()` to the end of the first line?